.github/workflows/main.yml

```diff
@@ -173,7 +173,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-12, windows-latest]
+        os: [ubuntu-latest, macos-12, windows-2022]
```

Comment:
I updated this while troubleshooting some debug messages. While it wasn't the source, I figured I would leave this pinned to the current image that `windows-latest` points to since we did this for `macos-latest` recently.